Time Master/script.js

```diff
@@ -0,0 +1,147 @@
+// 1 Time object basics
+// Date object to represent the current time
+const currentTime = new Date();
+
+// Extracting hours, minutes, and seconds from the Date object
+const hours = currentTime.getHours();
+const minutes = currentTime.getMinutes();
+const seconds = currentTime.getSeconds();
+
+// Logging the current time
+console.log(`Current Time: ${hours}:${minutes}:${seconds}`);
+

```

Comment:
Users do not access the console, so remove all console logs before pushing your code.